sim/common/cata/other_effects.go

```diff
@@ -825,6 +786,207 @@ func init() {
 			})
 		})
 	}
+
+	for _, version := range []ItemVersion{ItemVersionLFR, ItemVersionNormal, ItemVersionHeroic} {
+		labelSuffix := []string{" (LFR)", "", " (Heroic)"}[version]
+
+		vialItemID := []int32{77979, 77207, 77999}[version]
+		core.NewItemEffect(vialItemID, func(agent core.Agent) {
+			character := agent.GetCharacter()
+
+			actionID := core.ActionID{SpellID: []int32{109721, 107994, 109724}[version]}
+			minDmg := []float64{3568, 4028, 4546}[version]
+			maxDmg := []float64{5353, 6042, 6819}[version]
```

Comment:
So what do we do with [Vial of Shadows](https://www.wowhead.com/cata/item=77999/vial-of-shadows)? Simc has a 0.226, 0.3, 0.339 ap scaling here while the dbc is missing it atm.

Leaving the ap scaling out turns this trinket extremely lackluster and adding it is >1k more dps 😅 with ap scaling it does about as much dmg over 180s for a feral as Fetish does for an Arms warr which seems legit?

This comment from simc worries me though that it wasn't in the dbc even back then 🤷:
> // ICD, base damage, and ap coeff determined experimentally on heroic version. Assuming dbc has wrong base damage. Normal and LFR assumed to be changed by the same %

Thoughts @NerdEgghead @zoristaken @RobFalfa @1337LutZ ?

Reply:
I'd go for the underwhelming implementation with the increased proc rate according to the "nerf"/adjustment it got and add an item notice